src/pylexibank/commands/check.py

```diff
@@ -0,0 +1,51 @@
+from collections import Counter
+
+from clldutils.clilib import command  # , confirm, ParserError
+from pylexibank.commands.util import get_dataset
+from pylexibank.commands.misc import test
+from termcolor import colored
+
+
+@command()
+def check(args):
+    ds = get_dataset(args)
+
+    # validate
+    print(colored("Validating CLDF...", "green"))
+    ds.cldf.validate()
+
+    test(args)
+
+    # Check languages
+    print(colored("Checking Languages...", "green"))
+    for lang in ds.cldf['LanguageTable']:
+        if not lang['Glottocode']:
+            print(colored(
+                "Warning: Language '%s' is missing a glottocode" % lang['Name'],
+                "yellow"
+            ))
+        else:
+            found = ds.glottolog.languoid(lang['Glottocode'])
```

Comment:
Looking up every single glottocode this way may incur quite a performance penalty. Above - say - 100 languages, creating a lookup à la
```python
glottocodes = {l.id for l in ds.glottolog.languoids()}
```
would be faster (and for less than 100 languages at least only add a fixed amount (<10) of secs to execution).